### PR TITLE
New test for 11 & 12.

### PR DESCRIPTION
**Pull request by mcenerney1**
diag test for 11 & 12 created for metrics testing.

### CODE REVIEW

File: CMake/cdat_modules_extra/checkout_testdata.cmake [hunk 1]
```diff
@@ -146,7 +146,7 @@ endif()
 
 # 5) Update the remote branches available on the testdata repo.
 execute_process(COMMAND
-  "${GIT_EXECUTABLE}" fetch --update-shallow --depth=1
+  "${GIT_EXECUTABLE}" fetch --depth=1
```
Comment (aashish24) [inline CMake/cdat_modules_extra/checkout_testdata.cmake]: what didn't work here?

Comment (aashish24) [inline CMake/cdat_modules_extra/checkout_testdata.cmake]: @mcenerney1 could you please respond when you get a chance?

Comment (doutriaux1) [inline CMake/cdat_modules_extra/checkout_testdata.cmake]: @aashish24 I think on his mac with old github it doesn't work.Similar to #1355 also see #1427

**Comment by aashish24** [comment]
Okay. We need to update this branch now since it has merge conflict. @mcenerney1 are you planning on doing that.. or should I?

**Comment by doutriaux1** [comment]
@aashish24 I rebased and will test now

**Pull request merged**